src/Php84/bootstrap.php

```diff
@@ -38,3 +38,15 @@ function array_any(array $array, callable $callback): bool { return p\Php84::arr
 if (!function_exists('array_all')) {
     function array_all(array $array, callable $callback): bool { return p\Php84::array_all($array, $callback); }
 }
+
+if (!function_exists('mb_trim')) {
```

Comment:
Those `mb_*` function should be defined in the php84 polyfill only when mbstring is loaded, to avoid risks of breaking feature detection by having those mbstring functions without having others.
Projects wanting to use those functions without depending on the extension are expected to require the mbstring polyfill.
this is also what we did for the php74 polyfill.